x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/planner/LocalExecutionPlanner.java

```diff
@@ -266,11 +268,40 @@ else if (node instanceof OutputExec outputExec) {
             return planExchangeSink(exchangeSink, context);
         } else if (node instanceof MergeExec mergeExec) {
             return planMerge(mergeExec, context);
+        } else if (node instanceof RrfScoreEvalExec rrf) {
+            return planRrfScoreEvalExec(rrf, context);
         }
 
         throw new EsqlIllegalArgumentException("unknown physical plan node [" + node.nodeName() + "]");
     }
 
+    private PhysicalOperation planRrfScoreEvalExec(RrfScoreEvalExec rrf, LocalExecutionPlannerContext context) {
+        PhysicalOperation source = plan(rrf.child(), context);
+
+        int scorePosition = -1;
+        int forkPosition = -1;
+        int pos = 0;
+        for (Attribute attr : rrf.child().output()) {
+            if (attr.name().equals("_fork")) {
```

Comment:
nit: it would be nice if we could use (already available?) constants for `_fork` and `_score` strings.